### PR TITLE
Added bzip2 and libxcb dependencies to FFmpeg 3.x easyconfigs

### PR DESCRIPTION
**Pull request by FokkeDijkstra**
bzip2 and libxcb are pulled in by FFmpeg if available. Since both have easyconfigs, and are pulled in by other software as well, I suggest to include these dependencies.
Also replaced 'so' with SHLIB_EXT for the intel easyconfigs.

### CODE REVIEW

**Comment by FokkeDijkstra** [comment]
Should I make the same changes to the 2.8.x 2016a easyconfigs?

File: easybuild/easyconfigs/f/FFmpeg/FFmpeg-3.1.3-intel-2016b.eb [hunk 1]
```diff
@@ -14,7 +14,9 @@ source_urls = ['http://ffmpeg.org/releases/']
 dependencies = [
     ('NASM', '2.12.02'),
     ('zlib', '1.2.8'),
+    ('bzip2', '1.0.6'),
     ('x264', '20160614'),
+    ('libxcb', '1.11.1'),
```
Comment (boegel) [inline easybuild/easyconfigs/f/FFmpeg/FFmpeg-3.1.3-intel-2016b.eb]: @FokkeDijkstra please use the `X11` bundle here, which includes `libxcb`: `(('X11', '20160819'),`

**Comment by boegel** [comment]
@FokkeDijkstra makes sense to do this for the FFmpeg 2.8.x easyconfigs too, but you'll need to make sure that the easyconfigs for these deps are in place too (see failing Travis tests)

**Comment by FokkeDijkstra** [comment]
Made the changes in all 2016* configs, but not in older ones.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 7 out of 7 (7 easyconfigs in this PR)
node2601.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/f38f60451b798a5c6ee9c4299999c182 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 7 out of 7 (7 easyconfigs in this PR)
node2140.delcatty.os - Linux SL 6.8, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/5113d3152a8a10fee70cdcbb00b019f7 for a full test report.

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Going in, thanks @FokkeDijkstra!

**Pull request merged**